DiabloII.Application/Mappers/Suggestions/SuggestionDtoToHalLayer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using AutoMapper;
+using DiabloII.Application.Responses.Suggestions;
+using DiabloII.Domain.Extensions;
+
+namespace DiabloII.Application.Mappers.Suggestions
+{
+    public class SuggestionDtoToHalLayer : Profile
+    {
+        public SuggestionDtoToHalLayer() => CreateMap<SuggestionDto, SuggestionHalResponse>()
+            .Ignore(dto => dto.Votes)
+            .Ignore(dto => dto.Comments);
```

Comment:
Those 2 things should be handled by automapper, but it looks to be impossible to support the dependency injection...